mta-archive-builder/README.md

```diff
@@ -8,13 +8,13 @@ This image can be used to build SAP Multitarget Applications (MTA) containing Ja
 On a linux machine you can run 
 
 ```
-docker run -v `pwd`:/project --rm mta-archive-builder mtaBuild --version
+docker run -v ${PWD}:/project --rm mta-archive-builder mtaBuild --version
 ```
 
 This will execute the MTA archive builder and print its version information.
 
 ```
-docker run --rm -v `pwd`:/project -it mta-archive-builder:latest mtaBuild --mtar dummy.mtar --build-target NEO build
+docker run --rm -v ${PWD}:/project -it mta-archive-builder:latest mtaBuild --mtar dummy.mtar --build-target NEO build
```

Comment:
Does this work if the dir contains spaces? "${PWD}" might be safer

Reply:
No it does not :). Fixed.

Reply:
Just tried it and it worked locally. So maybe rather academical comment.